clumio/resource_organizational_unit.go

```diff
@@ -0,0 +1,307 @@
+// Copyright 2021. Clumio, Inc.
+
+// clumio_organizational_unit definition and CRUD implementation.
+package clumio
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	orgUnits "github.com/clumio-code/clumio-go-sdk/controllers/organizational_units"
+	"github.com/clumio-code/clumio-go-sdk/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// clumioOrganizationalUnit returns the resource for Clumio AWS Connection.
+
+func clumioOrganizationalUnit() *schema.Resource {
+	entitySchema := &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The Clumio assigned ID of the entity.",
+			},
+			"type": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				Description: "The entity type.",
+			},
+		},
+	}
+	return &schema.Resource{
+		// This description is used by the documentation generator and the language server.
+		Description: "Clumio AWS Connection Resource used to connect AWS accounts to Clumio.",
+
+		CreateContext: clumioOrganizationalUnitCreate,
+		ReadContext:   clumioOrganizationalUnitRead,
+		UpdateContext: clumioOrganizationalUnitUpdate,
+		DeleteContext: clumioOrganizationalUnitDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "OrganizationalUnit Id.",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"name": {
+				Description: "Unique name assigned to the organizational unit.",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"description": {
+				Description: "A description of the organizational unit.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"parent_id": {
+				Description: "The Clumio-assigned ID of the parent organizational unit" +
+					" under which the new organizational unit is to be created.",
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"users": {
+				Description: "List of user ids to assign this organizational unit.",
+				Type:        schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional: true,
+			},
+			"entities": {
+				Description: "List of user ids to assign this organizational unit.",
```

Comment:
Description copy-past error

Reply:
Similarly can `entities` be two lists of entities to add and remove like the REST API?